metricbeat/module/windows/perfmon/_meta/fields.yml

```diff
@@ -1,26 +1,18 @@
 - name: perfmon
   type: group
   fields:
-    - name: counters
+    - name: perfmon.counters
       type: group
       description: >
-        Grouping of different counters
-      fields: 
-        - name: group
+        Grouping of different counters   
+      fields:
+        - name: alias
```

Comment:
Is the alias and query going to be part of the event? Based on your example object above this is not the case. Means we don't need it in here? I think this file can actually be empty as the name is defined by the namespace.